modules/installation-osp-kuryr-config-yaml.adoc

```diff
@@ -41,25 +41,26 @@ networking:
   machineNetwork:
   - cidr: 10.0.0.0/16
   serviceNetwork:
-  - 172.30.0.0/16
+  - 172.30.0.0/16 <1>
   networkType: Kuryr
 platform:
   openstack:
     cloud: mycloud
     externalNetwork: external
     computeFlavor: m1.xlarge
     lbFloatingIP: 128.0.0.1
-    trunkSupport: true
-    octaviaSupport: true
+    trunkSupport: true <2>
+    octaviaSupport: true <2>
 pullSecret: '{"auths": ...}'
 sshKey: ssh-ed25519 AAAA...
 ----
-
-[NOTE]
-====
-Both `trunkSupport` and `octaviaSupport` are automatically discovered by the
+<1> Due to Octavia behavior the service subnet actually created by the
```

Comment:
Does something like this work for you?

> The Amphora Octavia driver creates two ports per load balancer. As a result, the service subnet that the installer creates is twice the size of the CIDR that is specified as the `serviceNetwork` property. The larger range is required to prevent IP address conflicts.

Reply:
Sure, I just updated the patch with the suggestion.